README.md

```diff
@@ -1,15 +1,60 @@
-# SSWG Logging API
 
-* Proposal: SSWG-xxxx
+# Proposal Review: SSWG-0001 (Server Logging API)
+
+After the great [discussion thread](https://forums.swift.org/t/server-logging-api/18834), we are proposing this as a final revision of this proposal and enter the proposal review phase which will run until the 20th January 2019.
+
+We have integrated most of the feedback from the discussion thread so even if you have read the previous version, you will find some changes that you hopefully agree with. To highlight a few of the major changes:
+
+- Logging meta-data is now structures and now supports nested dictionaries and list values, apart from strings of course.
```

Comment:
apart from == except; should this be "in addition to" ?

"now structures and now" -> "now structures and, "